rpc/api/args_test.go

```diff
@@ -932,15 +932,6 @@ func TestCallArgsNotStrings(t *testing.T) {
 	}
 }
 
-func TestCallArgsToEmpty(t *testing.T) {
```

Comment:
This test could be left in by changing the check that `from` is populated correctly and `to` is nil